kmm/kmm-feed/src/commonMain/kotlin/com/gchristov/newsfeed/kmmfeed/FeedViewModel.kt

```diff
@@ -92,7 +92,7 @@ class FeedViewModel(
             try {
                 val feedUpdate = getSectionedFeedUseCase(
                     pageId = nextPage,
-                    feedQuery = "brexit,fintech",
+                    feedQuery = state.value.searchQuery,
```

Comment:
Nice, this means that searching should actually be showing updated results, right?

Reply:
Need to check this when the API is integrated, but seems like the `state` picks up changes like a `@Binding` as I understand it...

Reply:
API already supports searching, just until now we were only technically searching for “brexit,fintech”. 

To finalise this we’d have to call loadContent() when we’re updating the search string in the state.

Reply:
This is pending too